waterlog/src/Components/Information.js

```diff
@@ -18,23 +20,30 @@ export default class InformationComponent extends Component {
 		};
 	}
 	render() {
-		return (
-			<div className="card bg-light container infomation p-2" style={{ borderRadius: 0, marginTop: '15px', marginBottom: '15px' }}>
-				<h5 className="card-title text-center">Wastage</h5>
-				<div className="card-body">
-					<div className="row">
-						<div className="text-center right information">
-							<h6>Money lost</h6>
-							<p> R {this.state.moneyLost}</p>
-						</div>
-						<div className="text-center information">
-							<h6> Total water lost</h6>
-							<p> {this.state.waterLost} l/hour</p>
-						</div>
-					</div>
-					<ProgressBarComponent percentage={this.state.percentage} />
-				</div>
-			</div>
+		return ( 
+		<div className = "card bg-light container infomation p-2"
+			style = {
```

Comment:
Make use of an external file for your styling.
Inline css takes precedence in the order of styling.

Reply:
updated